.github/workflows/ci.yml

```diff
@@ -27,6 +27,9 @@ jobs:
           yarn clean
           yarn build
           yarn test
+      - name: Bundle AWS SDKv2
+        run: |
+          yarn --frozen-lockfile --production --modules-folder ./target/node_modules
```

Comment:
ideally we'd use `esbuild` (or even `ncc`) to bundle the lambda logic into a single JS file (e.g. https://github.com/guardian/pinboard/blob/a11aa0c51ad795ca34b5ec87621202ee8f161069/archiver-lambda/package.json#L8) first performing a type check with `tsc --noEmit`

Reply:
fair enough. This only includes aws-sdk v2 and its dependencies though, because `devDependencies` are excluded from installation here.

This line is coming out in the update-to-aws v3 PR though, so maybe it's OK to keep in for the time being?